docs/runtime/compiler-ir.md

```diff
@@ -29,6 +29,9 @@ enso --no-compile-dependencies --no-ir-caches --vm.D enso.compiler.RemoveUnusedI
 
 ## Dumping IR
 
+Note that to properly dump module from standard library, **it is recommended to
+run with --no-ir-caches**.
```

Comment:
As this PR will modify the `.ir` and `.bindings` files significantly, it should [change the checksum header](https://github.com/enso-org/enso/blob/9390adb60ecdf69112b4a84b2d87e108f6adc3fb/docs/runtime/ir-caching.md#portability-and-versioning) to avoid mixing the new caches with those from other branches.

Reply:
Good idea. Done in 6cce54fbaea07754dd51eab315e4b62bedd86703